e2e-tests/gef/cypress/e2e/facility/unissued-facilities/unissued-facilities-bank-review-date.spec.js

```diff
@@ -0,0 +1,159 @@
+                facilityOneId = facility.body.details._id;
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_ONE);
+              });
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_TWO),
+              );
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CONTINGENT, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_THREE),
+              );
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_FOUR),
+              );
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_FOUR),
+              );
+              cy.apiSetApplicationStatus(dealId, token, CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED);
+            });
+          });
+        });
+    });
+
+    beforeEach(() => {
+      cy.saveSession();
+      cy.login(BANK1_MAKER1);
+    });
+
+    describe('Visiting from unissued facilities page', () => {
+      beforeEach(() => {
+        cy.visit(relative(`/gef/application-details/${dealId}/unissued-facilities`));
+        unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+        aboutFacilityUnissued.issueDateDay().type(dateConstants.threeDaysDay);
+        aboutFacilityUnissued.issueDateMonth().type(dateConstants.threeDaysMonth);
+        aboutFacilityUnissued.issueDateYear().type(dateConstants.threeDaysYear);
+
+        aboutFacilityUnissued.shouldCoverStartOnSubmissionNo().click();
+        aboutFacilityUnissued.coverStartDateDay().type(dateConstants.threeDaysDay);
+        aboutFacilityUnissued.coverStartDateMonth().type(dateConstants.threeDaysMonth);
+        aboutFacilityUnissued.coverStartDateYear().type(dateConstants.threeDaysYear);
+
+        aboutFacilityUnissued.coverEndDateDay().type(dateConstants.threeMonthsOneDayDay);
+        aboutFacilityUnissued.coverEndDateMonth().type(dateConstants.threeMonthsOneDayMonth);
+        aboutFacilityUnissued.coverEndDateYear().type(dateConstants.threeMonthsOneDayYear);
+
+        aboutFacilityUnissued.isUsingFacilityEndDateNo().click();
+
+        aboutFacilityUnissued.continueButton().click();
+      });
+
+      it('should redirect user to the about unissued facility page when clicking back link', () => {
+        cy.url().should('eq', relative(`/gef/application-details/${dealId}/unissued-facilities/${facilityOneId}/bank-review-date`));
+
+        bankReviewDate.backLink().click();
+
+        cy.url().should('eq', relative(`/gef/application-details/${dealId}/unissued-facilities/${facilityOneId}/about`));
+      });
+
+      it('should display error messages', () => {
```

Comment:
this could go in a describe explaining why there are error messages, eg `describe('when the date is entered in the incorrect format', () => {...`